packages/backend/migrations/20230202124709_create_ledger_transfers_table.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('ledgerTransfers', function (table) {
```

Comment:
Tigerbeetle transfers also have a `code` (type)
https://github.com/tigerbeetledb/tigerbeetle/blob/main/docs/DESIGN.md#events
I don't we're currently utilitzing it, _yet_
- https://github.com/interledger/rafiki/issues/202

Reply:
Maybe it's just optional `type` with DEPOSIT, WITHDRAWAL? Otherwise, just a normal "transfer"

Reply:
Yeah, I think combining that with `ledgerAccounts.type` of the `debitAccountId`/`creditAccountId` _should_ give us all the info we'd want